lib/health_cards/health_card.rb

```diff
@@ -56,14 +56,33 @@ def compress_payload(payload)
       # @param attributes [Array] An array of string with the attribute names that will be passed through
       #  when data is minimized
       def allow(klass, attributes)
-        resource_type = klass.name.split('::').last
-        allowable[resource_type] = attributes
+        allowable[klass] = attributes
```

Comment:
Since we permit inheriting these, should `allow/disallow` concatenate rather than overwrite?

If so, I'd initialize them with `Hash.new { |hash, key| hash[key] = [] }` rather than just `{}`, and then update `allow/disallow` to be `[dis]allowable[klass].concat(attributes)`.

Reply:
+1 for supporting concatenation

Reply:
I think this is a good idea, but I'd say it's out of scope for this PR. I can create a ticket with a few of the suggested improvements (concatenation of allow/disallow rules, use inheritance in allow/disallow rules) and work on that later.